spec/requests/legalaid_application_spec.rb

```diff
@@ -59,3 +50,44 @@
     end
   end
 end
+
+RSpec.describe 'create Legal aid applications with or without proceedings' do
+  let(:response_json) { JSON.parse(response.body) }
+
+  describe 'POST /v1/applications' do
+    it 'returns an application reference' do
+      post '/v1/applications'
+      expect(response_json['data']['attributes']['application_ref']).not_to be_empty
+      expect(response.status).to eql(201)
+      expect(response.content_type).to eql('application/json')
+    end
+
+    context 'when proceedings are saved along with application reference' do
+      let(:proceeding_type_first) { ProceedingType.create(code: 'PR0001', ccms_code: 'PBM23', meaning: 'test meaning1', description: 'test desc1') }
+      let(:proceeding_type_second) { ProceedingType.create(code: 'PR0002', ccms_code: 'PBM24', meaning: 'test meaning2', description: 'test desc2') }
+
+      it 'creates a new legal aid application without proceeding_type_codes ' do
```

Comment:
**[not a blocker]** ideally these descriptions should be fully human readable, so we should avoid using variable names in it and just use its human readable form.

Reply:
let's leave it for now.